techniques/failures/F46.html

```diff
@@ -84,8 +85,9 @@
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
-            <li>If any check above is false, then this failure condition applies
-                                    and the content fails the Success Criterion.</li>
+            <LI>If #1 is true, then this failure condition does not apply.</LI>
+            <li>If #1 is false, and any subsequent step is false, then this failure condition applies
```

Comment:
If there are headings in a table that is marked presentational, what do the browsers expose to AT about that? I assume they don't convey table semantics in that case.

Reply:
I would assume that was well - but I'm not clear what you are saying needs to be updated.

Reply:
I guess I'm finding the construction confusing. Perhaps we just have one expected result which would be:

If 1 and either 2 or 3 are false, then the failure condition applies.

Reply:
> If there are headings in a table that is marked presentational, what do the browsers expose to AT about that? I assume they don't convey table semantics in that case.

Yes, a screen reader will skip over a header labeled role-presentational. However, if a table has headers, that makes it a data table and now the rules for data tables apply.